src/PhpSpec/Extensions/DefaultMatchers/Extension.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace PhpSpec\Extensions\DefaultMatchers;
+
+use PhpSpec\Factory\ReflectionFactory;
+use PhpSpec\Matcher;
+use PhpSpec\ServiceContainer;
+use PhpSpec\ServiceContainer\IndexedServiceContainer;
+
+class Extension implements \PhpSpec\Extension
+{
+    /**
+     * @param ServiceContainer $container
+     * @param array $params
+     */
+    public function load(ServiceContainer $container, array $params)
+    {
+        $container->define('matchers.identity', function (IndexedServiceContainer $c) {
+            return new Matcher\IdentityMatcher($c->get('formatter.presenter'));
+        }, ['matchers']);
+        $container->define('matchers.comparison', function (IndexedServiceContainer $c) {
+            return new Matcher\ComparisonMatcher($c->get('formatter.presenter'));
+        }, ['matchers']);
+        $container->define('matchers.throwm', function (IndexedServiceContainer $c) {
+            return new Matcher\ThrowMatcher($c->get('unwrapper'), $c->get('formatter.presenter'), new ReflectionFactory());
```

Comment:
Since `ReflectionFactory` parameter was removed from constructor it can be removed here too.